src/Arg.roc

```diff
@@ -55,14 +56,14 @@ Parser a := [
     Lazy ({} -> a),
 ]
 
-## Indices in an arguments list that have already been parsed.
+# Indices in an arguments list that have already been parsed.
 Taken : Set Nat
 
-## A representation of parsed and unparsed arguments in a constant list of
-## command-line arguments.
-##
-## Used only internally, for efficient representation of parsed and unparsed
-## arguments.
+# A representation of parsed and unparsed arguments in a constant list of
```

Comment:
Do we not want these to be in docs anymore? (I'm wondering about the `##` to` #` change)

Reply:
Yeah, this type is not exposed so nothing is generated in documentation. I was thinking this would make the intent that this is an internal comment clearer.